src/Examine/Search/IQuery.cs

```diff
@@ -17,8 +17,9 @@ public interface IQuery
         /// the provider can still handle it.
         /// </remarks>
         /// <param name="query">The query.</param>
+        /// <param name="loadedFieldNames">The fields to load in the result set.</param>
         /// <returns></returns>
-        IBooleanOperation NativeQuery(string query);
+        IBooleanOperation NativeQuery(string query,ISet<string> loadedFieldNames = null);
```

Comment:
optional parameters are binary breaking changes. These will need to be overloads.